src/helpers/validation.ts

```diff
@@ -74,6 +80,30 @@ export function validateForm(
     validate: () => true
   });
 
+  ajv.addFormat('twitter-handle', {
+    validate: (value: string) => {
+      if (!value) return false;
+
+      return !!value.match(/^[a-zA-Z0-9_]+$/);
+    }
+  });
+
+  ajv.addFormat('github-handle', {
+    validate: (value: string) => {
+      if (!value) return false;
+
+      return !!value.match(/^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$/);
+    }
+  });
+
+  ajv.addFormat('discord-handle', {
+    validate: (value: string) => {
+      if (!value) return false;
+
+      return !!value.match(/^[a-zA-Z0-9_]+$/);
```

Comment:
I found the allowed format, we should support alpha numeric + dash and max 25 chars.
<img width="686" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/16245250/86cc1c12-d9b2-47fb-a72a-6e7c773b88bf">
This works also for invite code, I suggested some change in the placeholder and error message.